### PR TITLE
ICSP and IDMS interoperability release note

### PR DESCRIPTION
**Pull request by mburke5678**
Release note for [OCPSTRAT-898 - Need to ensure ICSP and IDMS interoperate especially in multi-tenant scenario ](https://github.com/openshift/openshift-docs/pull/70881)

### CODE REVIEW

**Comment by mburke5678** [comment]
@cbippley @obrown1205 @dfitzmau PTAL

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -579,6 +579,16 @@ With this release, you can monitor updates for individual nodes as a Technology
 [id="ocp-4-15-nodes"]
 === Nodes
 
+[id="ocp-4-15-nodes-icsp-idms-compatibility"]
+==== ICSP, IDMS, and ITMS are now compatible
+
+`ImageContentSourcePolicy` (ICSP), `ImageDigestMirrorSet` (IDMS), and `ImageTagMirrorSet` (ITMS) objects now function in the same cluster at the same time. Previously, in order to use the newer IDMS or ITMS objects, you needed to delete any ICSP objects. Now, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring].
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:585]:
```suggestion
`ImageContentSourcePolicy` (ICSP), `ImageDigestMirrorSet` (IDMS), and `ImageTagMirrorSet` (ITMS) objects now function in the same cluster at the same time. Previously, to use the newer IDMS or ITMS objects, you needed to delete any ICSP objects. Now, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring].
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -579,6 +579,16 @@ With this release, you can monitor updates for individual nodes as a Technology
...
+
+[IMPORTANT]
+====
+Using an ICSP object to configure repository mirroring is a deprecated feature. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```
Comment (dfitzmau) [inline release_notes/ocp-4-15-release-notes.adoc:589]: Will/might , "Might" would tie us in less knots if a feature were somehow, although rare, brought back to life.

**Comment by dfitzmau** [comment]
> @cbippley @obrown1205 @dfitzmau PTAL

Thanks, @mburke5678 . From a release note perspective, the note looks good to me. I see a table entry exists as reflects the feature accurately. 

Screenshot from 2024-02-14 08-43-16

File: release_notes/ocp-4-15-release-notes.adoc (hunk 2 above)
Comment (mburke5678) [reply release_notes/ocp-4-15-release-notes.adoc:589]: @dfitzmau Good thought, but "will" seems to be the standard wording in our docs. Likely would want to do a global find/replace...

**Pull request merged**